trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -60,16 +68,16 @@ describe('controllers - case', () => {
             status: 200,
             data: mockAmendments,
           });
-      });
 
-      it('should render deal template with data', async () => {
-        const req = {
+        req = {
           params: {
             _id: mockDeal._id,
           },
           session,
```

Comment:
Do we need a user session to ensure only `PIM` user can action this command. Thanks